crates/api/src/kitsune.rs

```diff
@@ -57,6 +57,15 @@ pub type DynKitsuneHandler = Arc<dyn KitsuneHandler>;
 
 /// The top-level Kitsune2 api trait.
 pub trait Kitsune: 'static + Send + Sync + std::fmt::Debug {
+    /// Register the kitsune handler. This should be called exactly once,
+    /// before any other apis are invoked on the instance. The only reason
+    /// this isn't in the factory is to support a struct containing a
+    /// kitsune instance as well as acting as its handler.
```

Comment:
Should we document that implementations of this trait should error if called more than once?

Reply:
Done: https://github.com/holochain/kitsune2/pull/135/commits/6c5610ff0a986bcb00aaaab26f26c3438b8db4cc